### PR TITLE
added assertpy and set up venv

### LINKED ISSUE #5: Add assertpy.
## User Story
We want to have easy ways to write tests.

## Proposed Solution
Install assertpy which supports fluent apis and has many pre-defined assertions.

### PR DESCRIPTION
**Pull request by CorningTopTier**
Fixes #5

### CODE REVIEW

File: myenv/bin/Activate.ps1 [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+Pushes the python executable for a virtual environment to the front of the
+$Env:PATH environment variable and sets the prompt to signify that you are
+in a Python virtual environment. Makes use of the command line switches as
+well as the `pyvenv.cfg` file values present in the virtual environment.
+
+.Parameter VenvDir
+Path to the directory that contains the virtual environment to activate. The
+default value for this is the parent of the directory that the Activate.ps1
+script is located within.
+
```
Comment (CorningTopTier) [inline myenv/bin/Activate.ps1:15]: Add comment here. Blah Blah blah

YOURE WRONG

File: myenv/bin/Activate.ps1 [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+        Remove-Item -Path env:VIRTUAL_ENV_PROMPT
+    }
+
+    # Just remove the _PYTHON_VENV_PROMPT_PREFIX altogether:
+    if (Get-Variable -Name "_PYTHON_VENV_PROMPT_PREFIX" -ErrorAction SilentlyContinue) {
+        Remove-Variable -Name _PYTHON_VENV_PROMPT_PREFIX -Scope Global -Force
+    }
+
+    # Leave deactivate function in the global namespace if requested:
+    if (-not $NonDestructive) {
```
Comment (CorningTopTier) [inline myenv/bin/Activate.ps1:110]: Wrong here too

**Review by CorningTopTier — COMMENTED** [review]
Did 2 things wrong

**Review by mkienenb — APPROVED** [review]
Looks good to me.

**Pull request merged**